changelogs/fragments/33-fix-boot-ssh-key-setting.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - bugfix for incorrect handling of ssh-keys for the community.hcloud.boot module. Trying to set the boot record with the module lets the API return errors (reported in https://github.com/ansible-collections/community.hrobot/issues/32). This is and the idempotence for ssh-keys is fixed with the https://github.com/ansible-collections/community.hrobot/pull/33 MR.
```

Following the exact format explained [here](https://docs.ansible.com/ansible/devel/community/development_process.html#changelog-fragment-entry-format):
```suggestion
  - boot - fix incorrect handling of ssh-keys (https://github.com/ansible-collections/community.hrobot/issues/32, https://github.com/ansible-collections/community.hrobot/pull/33).
```